cmd/cluster/kubevirt/create_test.go

```diff
@@ -0,0 +1,69 @@
+package kubevirt
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	apifixtures "github.com/openshift/hypershift/api/fixtures"
+	hyperv1 "github.com/openshift/hypershift/api/hypershift/v1beta1"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+)
+
+func Test_ApplyPlatformSpecificValues(t *testing.T) {
+	tests := map[string]struct {
+		argsOpts             core.CreateOptions
+		expectedPlatformOpts apifixtures.ExampleOptions
+		expectedError        bool
+	}{
+		"when networks is configured": {
```

Comment:
don't we need an extra test for when networks are omitted ? i.e. assert the default network is always present.

Reply:
Those test corresponde to the nodepool, if there is no networks at the nodepool the controller will add just the default one, I will add a test there.

Reply:
Ahh wait is already implicitly tested since the kubevirt_test.go at nodepool_controller already check that the default network is set if no networks are passed.